.github/workflows/cd.yml

```diff
@@ -9,9 +9,19 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: DerYeger/pnpm-setup-action@master
+      - uses: actions/checkout@v3
+
+      - uses: pnpm/action-setup@v2
+        name: Install pnpm
+        with:
+          version: 8.2.0
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
```

Comment:
I think with this, we can avoid setting `NODE_AUTH_TOKEN` by adding `registory-url` here. Let's do that. (It can be done in a follow-up PR.)

Reply:
Not sure I understand what you're talking about :)

Reply:
I'll add some commits.